posix/netdb.c

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/netdb.h>
+#include <openenclave/corelibc/stdlib.h>
+#include <openenclave/corelibc/sys/socket.h>
+#include <openenclave/internal/posix/raise.h>
+#include <openenclave/internal/posix/resolver.h>
+#include <openenclave/internal/thread.h>
+#include <openenclave/internal/trace.h>
+
+static size_t _resolver_table_len = 3;
+static oe_resolver_t* _resolver_table[3] = {0}; // At most 3
+static oe_spinlock_t _lock = OE_SPINLOCK_INITIALIZER;
+
+/* Called by the public oe_load_module_host_resolver() function. */
+int oe_register_resolver(int resolver_priority, oe_resolver_t* presolver)
```

Comment:
>presolver [](start = 63, length = 9)

Style: do not prefix pointer variables with 'p', particularly in this case where it results in a different English word.

Reply:
Absolutely agree. I have been fixing these as I see them. I'll do a scrub.